dev/conformance/runner.ts

```diff
@@ -16,7 +16,8 @@
 
 import {expect} from 'chai';
 const duplexify = require('duplexify');
-import googleProtoFiles from 'google-proto-files';
+const googleProtoFiles = require('google-proto-files');
```

Comment:
I needed to change this to run the tests locally. To make sure that we catch this type of issue automatically, I added the conformance tests back to the CI.